src/styles/shared/_buttons.scss

```diff
@@ -235,13 +236,16 @@
     }
 
     &:focus {
-      box-shadow: 0 0 0 border-width('thick') var(--p-border-secondary);
-      @include no-focus-ring;
+      box-shadow: 0 0 0 border-width('base') var(--p-border-secondary);
```

Comment:
This was probably out of date and changed in figma, or an oversight. This is the proper border width